### PR TITLE
Fix styling for Send: Sign TX and TxBroadcasted [ch4024]

### PR DESCRIPTION
**Pull request by GuillaumeRx**
* Added Asset Icon next to Sent amount

* Created LinkOut Component for TxID

* Added color props to Button

* Various Style corrections

Story details: https://app.clubhouse.io/mycrypto/story/4024

### CODE REVIEW

File: common/v2/components/LinkOut.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+interface Props {
+  text: string;
+  link: string;
+  truncate?(text: string): string;
+}
+
+const LinkOut = ({ text, link, truncate }: Props) => {
+  const content = truncate ? truncate(text) : text;
+  return (
+    <LinkOutWrapper href={link} target="_blank">
```
Comment (vilsbole) [inline common/v2/components/LinkOut.tsx]: Every `target="_blank">` should be accompanied with `noreferrerr`  & `nooopener`
https://developers.google.com/web/tools/lighthouse/audits/noopener

I'm for merging all our different link components into a single one.
We can continue discussing on slack

File: common/v2/components/Button.tsx [hunk 2]
```diff
@@ -4,15 +4,21 @@ import { Button } from '@mycrypto/ui';
...
+    props.fullwidth === true &&
+    css`
+      width: 100%;
+      margin-top: 1rem;
+    `}
+  ${props =>
+    props.color &&
+    css`
+      color: ${props.color};
+    `}
```
Comment (vilsbole) [inline common/v2/components/Button.tsx]: For a single property it's shorter to write:
```
color: ${props => props.color ? props.color : inherit }
```

**Comment by vilsbole** [comment]
FYI: 
- Prefer rebase over merge. It's more respectful for the other devs (master is the source of truth), and generates a more readable history.

File: common/v2/components/LinkOut.tsx (hunk 1 above)
Comment (GuillaumeRx) [reply common/v2/components/LinkOut.tsx]: Thanks for the info, gonna add this asap !

Let's discuss ont this.

**Comment by vilsbole** [comment]
Update is missing `noopenner` atrr.
Merging in favour of refactoring of link components

**Comment by GuillaumeRx** [comment]
According to : https://developers.google.com/web/tools/lighthouse/audits/noopener

> rel="noreferrer" attribute has the same effect [as rel="noopener"], but also prevents the Referer header from being sent to the new page. See Link type "noreferrer".

Should I add it also ?

**Comment by vilsbole** [comment]
Sorry my mistake. Good call!

**Pull request merged**